sfs/mono/esaedge.py

```diff
@@ -0,0 +1,400 @@
+"""Compute driving functions for various systems.
+
+.. include:: math-definitions.rst
+
+.. plot::
```

Comment:
None of this code is used in this file, it should be completely removed.

Reply:
da37636be625e9dfbb5fa98ceee3b2e336037006